README.md

```diff
@@ -170,6 +171,10 @@ module.exports = {
 
 Jest can be used in projects that use [webpack](https://webpack.js.org/) to manage assets, styles, and compilation. webpack does offer some unique challenges over other tools. Refer to the [webpack guide](https://jestjs.io/docs/en/webpack) to get started.
 
+### Using parcel
+
+Jest can be used in projects that use [parcel-bundler](https://parceljs.org/) to manage assests, styles, and compilation similar to webpack. Parcel bundler offers blazing fast performance utilizing multicore processing, and requires zero configuration.
```

Comment:
We don't need the "pitch" for parcel, methinks. Is there no parcel config (or vice versa) needed to work with jest?

Reply:
yes we only need to add dependencies and we are good to go parcel is faster than webpack

Reply:
I'm fine with adding parcel to the list, and saying it works without config. But we don't need to say "blazing fast" etc - marketing should be up to Parcel, not us 🙂

Reply:
So I should remove that "blazing fast" right like as follows:

`Jest can be used in projects that use [parcel-bundler](https://parceljs.org/) to manage assets, styles, and compilation similar to webpack. The parcel-bundler utilizes multicore processing and requires zero configuration.`